mods/ts/sequences/aircraft.yaml

```diff
@@ -17,16 +17,17 @@ scrin:
 apache:
 	Inherits: ^VehicleOverlays
 	icon: apchicon
-	rotor: lrotor
-		Length: 4
+	rotor: harpyrotor
+		Start: 32
+		Length: 31
 		Offset: 0, 0, 16
 		ZRamp: 1
-	slow-rotor: lrotor
-		Start: 4
-		Length: 8
+		Tick: 25
+	slow-rotor: harpyrotor
+		Length: 31
 		Offset: 0, 0, 16
 		ZRamp: 1
-
+		Tick: 50
 orcatran:
```

Comment:
Style nit: Missing new line above this one.

Reply:
*below